backend/app/testbot/data_loader.py

```
@@ -19,18 +168,22 @@ def load_file(file_path: str) -> List[str]:
             return load_csv(file_path)
-        elif ext == '.xlsx':
+        elif ext == '.xlsx' or ext == '.xls':
             return load_excel(file_path)
+        elif ext == '.json':
+            return load_json(file_path)
+        elif ext == '.pdf':
+            return load_pdf(file_path)
         else:
             logger.warning(f"Unsupported file type: {file_path}")
             return []
     except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)